### ISSUE TITLE
Show download speed and stats during playback?

### ISSUE THREAD

**Issue by krugster** [OstlerDev/PopcornTV#193]
Sometimes playback gets stuck and it would be helpful to see if the download has stalled for some reason.  Perhaps this could be periodically printed out to the logs (every 10s or so)?

**Comment by OstlerDev** [comment]
Pretty complex for a few reasons, firstly that we would need to ping the server every 10 seconds to grab the progress, then update the DOM with the changes. Most likely something I will not implement because there are other more pressing issues (such as the Desktop Application).

In the future I may revisit this if it is:
- More requested
- Something that backers of the project desire
- I have some free time :)

**Comment by krugster** [comment]
Makes sense.  What about the ability to query it from popcorntv on demand?  Like a REST API that lets you externally dump the current stats of the system?  Presumably it's only ever downloading one torrent right?

**Comment by OstlerDev** [comment]
If you run PopcornTV in Debug mode it will show the full download speed in the Console. And yes, it should only be download one at a time (Unless Multiple Apple TV's are connected).

Just run it with ``node atv.js Debug`` be sure to add sudo to the start if on Mac.

**Comment by krugster** [comment]
oh cool, I'll give that a try.  What about an ability to dynamically turn on or off debug mode ;-) ?

**Comment by OstlerDev** [comment]
@krugster There is no way to do that dynamically without destroying and creating the logger again. Plus, seeing as there is no GUI there would not be a good way. This will be implemented in the Desktop Application.
